app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: [Player]) -> int:
+    return sum(member.get_rating() for member in team)
+
+
+def elves_concert(team: [Elf]) -> None:
```

Comment:
Consider renaming the variable from `team` to `elves_team` to make it more descriptive and indicative of the content it holds.